openmc/geometry.py

```diff
@@ -322,6 +322,19 @@ def get_all_cells(self):
         else:
             return OrderedDict()
 
+    def get_all_filled_cells(self):
+        """Return all cells that are contained within the universe that are
+        filled with a material
+
+        cells : collections.OrderedDict
+            Dictionary whose keys are cell IDs and values are :class:`Cell`
+            instances
+
+        """
+
+        return {id:cell for id, cell in self.root_universe.get_all_cells().items() if cell.fill is not None}
```

Comment:
Any reason not to use the new Universe.get_all_filled_cells() method here?

Reply:
Thanks for spotting, I shall change to use the new method, it shortens the line nicely